released/packages/coq-quickchick/coq-quickchick.1.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+name: "coq-quickchick"
+version: "1.0.1"
+maintainer: "llamp@seas.upenn.edu"
+
+homepage: "https://github.com/QuickChick/QuickChick"
+dev-repo: "https://github.com/QuickChick/QuickChick.git"
+bug-reports: "https://github.com/QuickChick/QuickChick/issues"
+license: "MIT"
+
+build: [ make "-j%{jobs}%" "plugin" ]
+install: [ make "install" ]
+remove: [ [ "rm" "-rf" "%{lib}%/coq/user-contrib/QuickChick" ]
+          [ "rm" "-f" "`which quickChick | grep opam`" ] ]
```

Comment:
Does it work using `["rm" "%{bin}%/quiChick"]` instead? This command may remove the wrong file.